src/pages/cli/reference/usage-data.mdx

```diff
@@ -7,7 +7,7 @@ export const meta = {
 
 At AWS, we develop and launch services based on what we learn from interactions with our customers. We use customer feedback to iterate on our product. Anonymized usage data helps us to better understand our customers’ needs, diagnose issues, and deliver features that improve the customer experience.
 
-Amplify CLI sends anonymized information such as system metadata, usage metrics and errors. The data is marked for deletion in a year from the point of it being collected. Amplify CLI does **not** collect personally identifiable information such as email addresses, usernames, keys, ARNs or any project - level information.
+Amplify CLI sends anonymized information such as system metadata, usage metrics and errors. The data is marked for deletion in a year from the point of it being collected. Amplify CLI does **not** collect personally identifiable information such as email addresses, usernames, keys, ARNs or project information such as names, arns and keys.
```

Comment:
```suggestion
Amplify CLI sends anonymized information such as system metadata, usage metrics and errors. The data is marked for deletion in a year from the point of it being collected. Amplify CLI does **not** collect personally identifiable information such as email addresses, usernames, keys, ARNs or project information such as names, ARNs, and keys.
```

updated with same ARN casing from earlier in the sentence and added an Oxford comma to align with the other pages. Quick note, "ARNs" is specified twice here in the personal information and project information. Is that intended?

Reply:
Ideally add an oxford comma to the first sentence as well: `such as system metadata, usage metrics, and errors.`